processor/src/main/java/org/mapstruct/ap/internal/model/source/Mapping.java

```diff
@@ -34,12 +34,16 @@
 import javax.lang.model.type.TypeKind;
 import javax.lang.model.type.TypeMirror;
 
+import org.mapstruct.ap.internal.model.common.Parameter;
 import org.mapstruct.ap.internal.model.common.TypeFactory;
+
```

Comment:
In my IDE it sorts based on the first package name. I don't know. All the other classes have all `org.mapstruct` grouped together.